report_a_breach/custom_upload_handler.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+from django.conf import settings
+from django.core.files.uploadedfile import TemporaryUploadedFile
+from django.core.files.uploadhandler import FileUploadHandler
+from django_chunk_upload_handlers.clam_av import (
+    FileWithVirus,
+    VirusFoundInFileException,
+)
+
+CHUNK_UPLOADER_RAISE_EXCEPTION_ON_VIRUS_FOUND = getattr(
+    settings,
+    "CHUNK_UPLOADER_RAISE_EXCEPTION_ON_VIRUS_FOUND",
+    False,
+)
```

Comment:
cleaner to use:
```suggestion
CHUNK_UPLOADER_RAISE_EXCEPTION_ON_VIRUS_FOUND = settings.CHUNK_UPLOADER_RAISE_EXCEPTION_ON_VIRUS_FOUND
```

Means that the default value for this setting (False) is found in the `base.py` settings file where all the other default values are found